### PR TITLE
Indix archiver

### PR DESCRIPTION
**Pull request by shrechak**
@ashwanthkumar @brewkode @ind9/analytics

### CODE REVIEW

File: dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala [hunk 1]
```diff
@@ -3,6 +3,7 @@ package com.indix.pail
 import java.io.IOException
 import java.util
 
+import _root_.util.DateHelper
```
Comment (ashwanthkumar) [inline dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala]: @shrechak Please change this without the `_root_`.

File: dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala [hunk 2]
```diff
@@ -138,4 +140,27 @@ object PailMigrateUtil {
...
 
+object IxPailArchiver {
+  val logger = Logger.getLogger(this.getClass)
+
+  def main(params: Array[String]) = {
+
+    val lastWeekBucket = DateHelper.weekInterval(new DateTime(System.currentTimeMillis()).minusDays(7))
+
+    val args = Args(params)
+    val baseInputDir = args("--base-input-dir")
```
Comment (ashwanthkumar) [inline dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala]: @shrechak I guess this should be `args("base-input-dir")` ?

Comment (shrechak) [inline dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala]: Oh yeah changing

File: dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala (hunk 1 above)
Comment (shrechak) [inline dfs-datastores/src/main/scala/com/indix/pail/PailMigrate.scala]: The Datehelper is under "util" package. Without __root__ is points to java.util package - as discussed.

**Comment by ashwanthkumar** [comment]
@shrechak LGTM

**Pull request merged**